src/common/AmazonS3Backend.ts

```diff
@@ -286,19 +286,11 @@ export class AmazonS3Backend implements IBackend {
   public updateFormData(folder: Folder, file: any, formData: any): Promise<any> {
     // append AWS upload key to the form data
     // needed in order to have a valid POST
-    let filePath = file.name;
```

Comment:
I think it would make sense to change this to use 'getFilePath' in the previous commit.

Reply:
That would make sense, but also since we are no longer using dropzonejs to submit the form for uploading files, this function never gets called, so I've added a commit to remove 'updateFormData'.